backend/routers/index.js

```diff
@@ -9,6 +9,32 @@ app.get('/', (req, res) => {
   res.status(200).send('Hellow world');
 });
 
+const myTroops = {
+  troops: [
+    {
+      id: 1,
+      level: 1,
+      hp: 1,
+      attack: 1,
+      defence: 1,
+      started_at: 12345789,
+      finished_at: 12399999,
+    }, {
+      id: 2,
+      level: 1,
+      hp: 1,
+      attack: 1,
+      defence: 1,
+      started_at: 12345789,
+      finished_at: 12399999,
+    },
+  ],
+};
+
+app.get('/kingdom/troops', (req, res) => {
+  res.status(200).send(myTroops);
+});
```

Comment:
May be we can write these handlers into separate files like this:
```js
const { Router } = require('express');
const router = Router();

function getTroops(req, res, next) {
  return res.send(myTroops);
}

// other handlers can be here

router.get('/', getTroops);

module.exports = router;
```

And then import these in app.js
```js
import { troops } from './routes';
// ... other middlewares
app.use('/troops', troops);
```

Reply:
yes please, great suggestion!

Reply:
instead of approving and just commenting, you can also ask for change during your review, do this more time! :)